frontend/src/container/NotificationWall.jsx

```diff
@@ -110,56 +113,73 @@ export class NotificationWall extends React.Component {
       interpolation: { escapeValue: false }
     }
 
-    const contentUrl = notification.content ? PAGE.WORKSPACE.CONTENT(notification.workspace.id, notification.content.type, notification.content.id) : ''
+    const isPublication = notification.content && notification.content.contentNamespace === CONTENT_NAMESPACE.PUBLICATION
+
+    const contentUrl = (
+      notification.content
+        ? (
+          isPublication
+            ? PAGE.WORKSPACE.PUBLICATION(notification.workspace.id, notification.content.id)
+            : PAGE.WORKSPACE.CONTENT(notification.workspace.id, notification.content.type, notification.content.id)
+        )
+        : ''
+    )
 
     if (entityType === TLM_ENTITY.CONTENT) {
+      const publicationIcon = isPublication ? 'fas fa-stream+' : ''
```

Comment:
Is `fas` required here as it is always in the other part of the icon's class?

Reply:
it's actually two icons with the `+`, so nope, we need it there